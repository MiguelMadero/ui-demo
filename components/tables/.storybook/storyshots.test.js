import initStoryshots from '@storybook/addon-storyshots';
import 'jest-styled-components';

// polyfill
window.URLSearchParams = class URLSearchParams {
  get() {}
};

initStoryshots({
  configPath: __dirname,
});
