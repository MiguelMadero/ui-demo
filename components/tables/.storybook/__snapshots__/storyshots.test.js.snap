// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Table default 1`] = `
.c4 {
  margin-left: -8px;
  margin-right: -8px;
  margin-top: -8px;
  margin-bottom: -8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c1 {
  background-color: #FFF;
}

.c3 {
  font-size: 16px;
  padding-left: 24px;
  padding-right: 24px;
  padding-top: 16px;
  padding-bottom: 16px;
  border-bottom-color: #DBDFE3;
  border-bottom-width: 1px;
  border-bottom-style: solid;
}

.c5 {
  padding-left: 8px;
  padding-right: 8px;
  padding-top: 8px;
  padding-bottom: 8px;
  width: 100%;
}

.c6 {
  padding-left: 8px;
  padding-right: 8px;
  padding-top: 8px;
  padding-bottom: 8px;
  width: 100%;
}

.c9 {
  font-size: 14px;
  padding: 8px;
}

.c2 {
  font-weight: 600;
}

.c8 {
  padding: 16px 24px;
  font-weight: 600;
}

.c0 {
  border: 1px solid #DBDFE3;
  margin-bottom: 16px;
  overflow: visible;
  border-radius: 2px;
}

.c7 {
  padding: 24px;
  border-bottom: 1px solid #DBDFE3;
}

.c7:last-of-type {
  border-bottom: none;
}

@media screen and (min-width:48em) {
  .c5 {
    width: 50%;
  }
}

@media screen and (min-width:48em) {
  .c6 {
    width: 25%;
  }
}

<div
  className="c0 c1 "
>
  <div
    className="c2 c3 "
  >
    <div
      className="c4 "
    >
      <div
        className="c5 "
      >
        <div
          className=""
        >
           Header 1 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           Header 2 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           Header 3 
        </div>
      </div>
    </div>
  </div>
  <div
    className="c7 "
  >
    <div
      className="c4 "
    >
      <div
        className="c5 "
      >
        <div
          className=""
        >
           1 1 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           1 2 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           1 3 
        </div>
      </div>
    </div>
  </div>
  <div
    className="c7 "
  >
    <div
      className="c4 "
    >
      <div
        className="c5 "
      >
        <div
          className=""
        >
           2 1 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           2 2 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           2 3 
        </div>
      </div>
    </div>
  </div>
  <div
    className="c8 c9 "
  >
    <div
      className="c4 "
    >
      <div
        className="c5 "
      >
        <div
          className=""
        >
           Footer 1 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           Footer 2 
        </div>
      </div>
      <div
        className="c6 "
      >
        <div
          className=""
        >
           Footer 3 
        </div>
      </div>
    </div>
  </div>
</div>
`;
